flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/codegen/ExprCodeGenerator.scala

```diff
@@ -215,7 +209,7 @@ class ExprCodeGenerator(ctx: CodeGeneratorContext, nullableInput: Boolean)
           input2Term.get,
           idx,
           nullableInput,
-          ctx.nullCheck)
+          true)
```

Comment:
This function doesn't need this boolean anymore?

Reply:
The `deepCopy` flag is not only passed as `ctx.nullCheck`, as far as I can see it has other usages as well, so I didn't want to touch it.